bash_completion

```diff
@@ -2341,6 +2369,12 @@ _comp_command_offset()
 #
 _comp_command()
 {
+    # We locally make the shell variable "words" in the unset state to tell
+    # "_comp_command_offset" that the index is intended to be that in
+    # "COMP_WORDS" instead of "words".
```

Comment:
```suggestion
    # We unset the shell variable `words` locally to tell
    # `_comp_command_offset` that the index is intended to be that in
    # `COMP_WORDS` instead of `words`.
```